src/wallet/wallettool.cpp

```diff
@@ -47,7 +47,7 @@ static void WalletCreate(CWallet* wallet_instance, uint64_t wallet_creation_flag
     wallet_instance->TopUpKeyPool();
 }
 
-static const std::shared_ptr<CWallet> MakeWallet(const std::string& name, const fs::path& path, const ArgsManager& args, DatabaseOptions options)
+static const std::shared_ptr<CWallet> MakeWallet(const std::string& name, const fs::path& path, const ArgsManager& args, DatabaseOptions options, CWallet::do_init_used_flag do_init_used_flag_val = CWallet::do_init_used_flag::Init)
```

Comment:
3ee478b9cba305dc87de1ad093ca14a89e58b516 it may be more maintainable and explicit to pass the value rather than use a default

Reply:
Generally, this should never be used. It's weird that the wallet tool wants to analyze the on-disk addressbook specifically.